src/main/java/org/amahi/anywhere/tv/fragment/ServerSelectFragment.java

```diff
@@ -134,8 +134,6 @@ private void setDefaultChecked() {
 
     private void setFalseChecked() {
         if (OPTION_CHECKED != null) {
-            setFalse();
```

Comment:
ok, that `.clear()` may do something ... no?

Reply:
`OPTION_CHECKED` is an ArrayList (All caps shouldn't be used for variable names though).
`.clear()` removes all elements from that ArrayList. 
However, earlier it could have possibly caused an issue because we were calling the `clear` function while incorrectly checking if `OPTION_CHECKED` is null or not.

Reply:
The meaning of the code afterwards is potentially not the same. It will be cleared *if* it's not `null` whereas before it was not.

Reply:
I have updated the code.